pkg/scheduler/api/node_info.go

```diff
@@ -607,3 +612,14 @@ func (ni *NodeInfo) getUnhealthyGPUs(node *v1.Node) (unhealthyGPUs []int) {
 	}
 	return
 }
+
+func (cs *CSINodeStatusInfo) Clone() *CSINodeStatusInfo {
+	newcs := &CSINodeStatusInfo{
+		CSINodeName:  cs.CSINodeName,
+		DriverStatus: make(map[string]bool),
+	}
+	for k, v := range newcs.DriverStatus {
```

Comment:
newcs is a empty map. Did you want to loop cs.DriverStatus?

Reply:
sorry for code mistake. it should be cs.DriverStatus